src/lib/y2partitioner/widgets/pages/lvm_vg.rb

```diff
@@ -95,13 +119,23 @@ def label
           _("Log&ical Volumes")
         end
 
+        def initial
```

Comment:
just curious: is it needed?

Reply:
Yes, this sets the tab as selected by default.